build/lib/locFunc.js

```diff
@@ -238,7 +238,7 @@ function refreshLangpacks() {
         }
         let packageJSON = JSON.parse(fs.readFileSync(path.join(locExtFolder, 'package.json')).toString());
         //processing extension fields, version and folder name must be changed manually.
-        packageJSON['name'] = packageJSON['name'].replace('vscode', textFields.nameText);
+        packageJSON['name'] = packageJSON['name'].replace('vscode', textFields.nameText).replace('pt-BR', 'pt-br');
```

Comment:
How about just lower-casing the name no matter what? Seems like we should just always expect the names to be lower cased